### PR TITLE
Andrea development - portfolio design and home page

### PR DESCRIPTION
**Pull request by andrea-ramirez**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,18 +2,34 @@
...
+            I am Computer Science student at Tecnológico de Monterrey from Venezuela 🇻🇪<br> 
+            Currently enrolled in the Bachelor of Computer Technologies Honors Program <br> 
+            🧚🏽‍♀️ In love with learning and working with others 💙 <br>
+            My greatest interests in tech are Mobile Development and Human Computer Interaction <br>                
+            In my free time I enjoy dancing and doing outdoor activities 🏞️ <br>
+            And while you are here you cand add me 😉
+            <!--Linkes of github + linkedin-->
+        </p>
+        <p>Click here for a random fun fact about me:</p>
+        <button onclick="addRandomGreeting()">Surprise me!</button>
```
Comment (sakshi-23) [inline portfolio/src/main/webapp/index.html:28]: You might want to change the name of this function as it no longer shows a greeting but a random fact

Comment (sakshi-23) [inline portfolio/src/main/webapp/index.html]: You might want to change the name of this function as it no longer shows a greeting but a random fact

File: portfolio/src/main/webapp/projects.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </div>
+    <h1>Andrea's Projects 🛠️</h1>
+    <h2 style="float: left;"> Punto Blanco Mobile App </h2>
+    <p style="float: left;">Developed a mobile app in Swift for Punto Blanco, Meditation and Wellness Center at ITESM, in a group of 3. <br>
+       App helps users follow guided meditations, record their performance and visualize their own data over a period of time <br>
+      In charge of implementation and maintenance of Firebase database</p>
+    <h2 style="float: left;"> Survive the Outbreak Videogame</h2>
+    <p style="float: left;">Created a three-level videogame in Java in a group of four. Similar to Pacman, designed to teach children healthy habits during quarantine. <br>
+        Led design of user interface, the theme and worked with Photoshop and Illustrator.</p>
+    <h2 style="float: left;">Database for Coronavirus patients </h2>
```
Comment (sakshi-23) [inline portfolio/src/main/webapp/projects.html]: In general it is a good idea to add all style elements like float:left in the css files

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,92 @@
...
+  margin-right: auto;
+  background-color: #007372;
+  border: none;
+  color: white;
+  padding: 10px 20px;
+  border-radius: 5%;
+  text-align: center;
+  font: geneva;
+  text-decoration: none;
+  display: inline-block;
```
Comment (sakshi-23) [inline portfolio/src/main/webapp/style.css]: You might want to remove this line "display: inline-block;" if your intention was to center align the button with the rest of the text

**Review by sakshi-23 — CHANGES_REQUESTED** [review]
Great work! Looks good overall. Left some comments!

File: portfolio/src/main/webapp/projects.html (hunk 2 above)
Comment (andrea-ramirez) [reply portfolio/src/main/webapp/projects.html]: I included those lines of code into a <div> tag with an id = "projects", and added the "float: left"  in the css file under that id. But it keeps going back to the center of the page when I run it
I was thinking I could create a different .css file, but I don't think that's the best option

Comment (andrea-ramirez) [reply portfolio/src/main/webapp/projects.html]: *into a div tag

Comment (sakshi-23) [reply portfolio/src/main/webapp/projects.html]: If you need it to left align add something like this:

.projects h2{
    text-align: left;
    margin: 1em;
}

.projects p{
   text-align: left;
    margin: 1em;
}

Basically currently your style is being overwritten by the default p & h2 properties. So basically here we need to add properties that are specific to your block.

Comment (andrea-ramirez) [reply portfolio/src/main/webapp/projects.html]: Thank you Sakshi! It did work :)
I was not noticing it was being overwritten

**Review by sakshi-23 — APPROVED** [review]

**Pull request merged**